### ISSUE TITLE
Downloading images crashes the software

### ISSUE THREAD

**Issue by sarvex** [Komet/MediaElch#976]
**Describe the bug**
Downloading images crashes the software

**To Reproduce**
```
MediaElch 2020-05-10 16:58:00.565 DEBUG : Entered, enabled= true
MediaElch 2020-05-10 16:58:00.565 DEBUG : Entered, enabled= false
MediaElch 2020-05-10 16:58:00.565 DEBUG : [FanartTv] Load movie data with image types: QUrl("https://webservice.fanart.tv/v3/movies/530915?api_key=842f7a5d1cc7396f142b8dd47c4ba42b")
MediaElch 2020-05-10 16:58:01.339 DEBUG : [NotificationBox] Hide Progress bar with id 20014
MediaElch 2020-05-10 16:58:01.339 DEBUG : Entered, id= 20014
MediaElch 2020-05-10 16:58:01.339 DEBUG : Entered, msg= "Downloading images..." id= 20014
MediaElch 2020-05-10 16:58:01.339 DEBUG : Entered, message= "Downloading images..." id= 20014
MediaElch 2020-05-10 16:58:01.340 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/3igGY57QsdJPTChto3ZBDT9vXTg.jpg")
MediaElch 2020-05-10 16:58:01.340 DEBUG : [DownloadManager] Start next download
MediaElch 2020-05-10 16:58:01.340 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/A3UGnYH7ESTBwEjWBFev7MhvBsB.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/y6OqnC0QJXwCzY9CthZm0ONv31B.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/3QQdJAMuXSBDK5uuTdgRNwJu1em.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/67HfnUIyYsxd11u99BHBFgFgCQf.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/qS6WwLJb8GlYcIV5Cj16J3RJDE3.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/z6wxnkqSTnzO1tcBui0ss7ehdm9.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/wz3MRiMmoz6b5X3oSzMRC9nLxY1.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/2qmA5mXnhA7Nelu2BTbJTvzq35u.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/61lXhIEZzSsT38yBro8Rh01GDmA.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/qASA1oFOwEvL7epg0HZkPE4SRwc.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/dCZSSEg3B8Kkax1uc5FyIhEW3Wm.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/Att2QAYBXyttQKCa6ZIoluWAvHt.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/fDEVG8sRsVuwWdF13Qm8FJT276v.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/xkuHCToqahrpzcFbqBlxIxbE6cV.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/cpHNAqOBP8SIcml1A8ODmtuEyNh.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/qhQbGDqm5W8frysaKJeNwJOkL1M.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/wvQImCMgCEApYHzFTmsYYwnCp4B.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/AuGiPiGMYMkSosOJ3BQjDEAiwtO.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("http://image.tmdb.org/t/p/original/2lBOQK06tltt8SQaswgb8d657Mv.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("https://assets.fanart.tv/fanart/movies/530915/hdmovielogo/1917-5d5f212d9f7f7.png")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("https://assets.fanart.tv/fanart/movies/530915/hdmovieclearart/1917-5e32ec776cfe9.png")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("https://assets.fanart.tv/fanart/movies/530915/moviedisc/1917-5e6b4362763df.png")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("https://assets.fanart.tv/fanart/movies/530915/moviebanner/1917-5e25f1c75530c.jpg")
MediaElch 2020-05-10 16:58:01.341 DEBUG : [DownloadManager] Enqueue download | QUrl("https://assets.fanart.tv/fanart/movies/530915/moviethumb/1917-5dd4acdf75278.jpg")
fish: “MediaElch” terminated by signal SIGSEGV (Address boundary error)
```

**Expected behavior**
Should download images

**Screenshots**
Not Applicable

**MediaElch Version:**
 - [ ] 2.6.7-dev (nightly)
 - [x] 2.6.6 stable

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [x] Linux (distribution: Manjaro Gnome)

**Additional context**

**Comment by bugwelle** [comment]
Hi,

were you downloading images for movies or TV shows or concerts?

What Qt version are you using? Could you paste the developer information here? See https://mediaelch.github.io/mediaelch-doc/contributing/bug-reports.html#developer-information

**Comment by sarvex** [comment]
MediaElch 2.6.6 - Ferenginar

Compiled with Qt version 5.14.2 (64 Release, GCC 9.3.0)
Using Qt version 5.14.2
System: Manjaro Linux (x86_64-little_endian-lp64)

Application dir: /usr/bin
Settings file: ~/.config/kvibes/MediaElch.conf
Data dir: ~/.local/share/kvibes/MediaElch
MediaInfo Version: MediaInfoLib - v19.09

**Comment by sarvex** [comment]
It is happening when I have a lot of movies (currently 1308 movies). 
It has started happening recently, earlier I was able to download. It tried cleaning the configuration directories but of no use.

**Comment by bugwelle** [comment]
hhmmm... You use the latest versions of Qt so it is an issue with MediaElch. I'm currently working on another change but will look into your issue after that. :)

**Comment by JonasCz** [comment]
I have this issue ("terminated by signal SIGSEGV" when downloading images) too, on Arch.

The AppImage version available here on GitHub doesn't have this issue, so I'm using that for now.

**Comment by bugwelle** [comment]
Hi,

I just wanted to inform you that I was able to reproduce this issue on Manjaro XFCE and that the bug will be fixed by #1078.
Turns out that this bug only happened with Kvantum installed which is why I wasn't able to reproduce this bug (I used Manjaro KDE for testing before...).

Regards,
Andre

**Comment by bugwelle** [comment]
#1078  is now merged. If you compile MediaElch yourself, check out the latest master. If not you will have to wait for the next release.

If this issue wasn't fixed for you, please let me know.
